caas/kubernetes/provider/modeloperator.go

```diff
@@ -693,6 +693,25 @@ func ensureExecRBACResources(objMeta meta.ObjectMeta, clock jujuclock.Clock, bro
 	role := &rbac.Role{
 		ObjectMeta: objMeta,
 		Rules: []rbac.PolicyRule{
+			{
+				APIGroups: []string{""},
```

Comment:
This change will disappear once rebase is done after https://github.com/juju/juju/pull/14699 merged